libs/algorithms/tests/unit/container_algorithms/generate_range.cpp

```diff
@@ -52,7 +77,7 @@ void test_generate(ExPolicy policy, IteratorTag)
 }
 
 template <typename ExPolicy, typename IteratorTag>
-void test_generate_async(ExPolicy p, IteratorTag)
+void test_generate_async(ExPolicy&& p, IteratorTag)
```

Comment:
@hkaiser  Could you please explain why the `&&`?

Reply:
In this case it doesn't really make a big difference, I just thought that using a forwarding reference here would be more idiomatic and avoids copying the execution policy.